src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] colors = {"red", "orange", "yellow", "green", "blue", "purple"};
+
+    public static String getRandomColor() {
```

Comment:
**Don’t use static methods in your solution**
Static methods are in general a bad practice. Let’s better create an instance of a class which method you want to call.

https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-static-methods-in-your-solution